client/src/components/commonComponents/Tabs/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, Fragment } from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Paper from '@material-ui/core/Paper';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './style';
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="textSecondary" align="center">
+      <Link color="inherit" href="https://www.urbannatural.com/">
+        Urban natural
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  );
+}
+
+const tabs = ['Contact Info', 'Questions', 'Book'];
+
+function getTabContent(tab) {
+  switch (tab) {
+    case 0:
+      // TODO: each return must call its relevant tab page component
+      return <>Contact Info </>;
+    case 1:
+      return <>Questions </>;
+    case 2:
+      return <>Booking </>;
+    default:
+      throw new Error('Unknown tab');
+  }
+}
+
+const Checkout = () => {
+  const classes = useStyles();
+  const [activeTab, setActiveTab] = useState(0);
+  const handleNext = () => {
+    setActiveTab(activeTab + 1);
+  };
+  const handleBack = () => {
+    setActiveTab(activeTab - 1);
+  };
+  return (
+    <Fragment>
+      <CssBaseline />
+      <AppBar position="absolute" color="default" className={classes.appBar}>
+        <Toolbar>
+          <Typography variant="h6" color="inherit" noWrap>
+            Urban Natural
+          </Typography>
+        </Toolbar>
+      </AppBar>
+
+      <main className={classes.layout}>
+        <Paper className={classes.paper}>
+          <Typography component="h1" variant="h4" align="center">
+            Checkout
```

Comment:
Can you give it another heading than 'checkout'

Reply:
yes, I put that into consideration too. I tried to change it to "Reservation" but it wasn't convenient with the page. Deleting it was not a very good choice either, so I will think of a better header when we combined it with the NavBar